library/core/src/main/java/com/google/android/exoplayer2/upstream/ParsingLoadable.java

```diff
@@ -65,7 +66,8 @@
    */
   public static <T> T load(DataSource dataSource, Parser<? extends T> parser, Uri uri, int type)
       throws IOException {
-    ParsingLoadable<T> loadable = new ParsingLoadable<>(dataSource, uri, type, parser);
+    // TODO: What headers should be used here? This is only ever called from a test
```

Comment:
Here's one place I wasn't sure what the purpose of it was, so I wasn't sure how to handle it.